crates/bitwarden-wasm-internal/Cargo.toml

```diff
@@ -18,6 +18,7 @@ crate-type = ["cdylib"]
 bitwarden-core = { workspace = true, features = ["wasm", "internal"] }
 bitwarden-crypto = { workspace = true, features = ["wasm"] }
 bitwarden-error = { version = "1.0.0", path = "../bitwarden-error" }
+bitwarden-send = { workspace = true, features = ["wasm"] }
```

Comment:
@coroiu this is needed to ensure send has `wasm-bindgen` as a dependency.

When compiling the whole project `bitwarden-error` has wasm enabled but `bitwarden-send` doesn't have the `wasm-bindgen` dependency which causes it to be compiled using the wasm variant. Ideally we would get it to use the symlinked wasm-bindgen